pkg/busybox

```diff
@@ -48,14 +48,17 @@ if [ "$DEBUGBUILD" = "1" ] ; then
 	    -i config.stage1
 fi
 
-make KCONFIG_ALLCONFIG=config.stage1 allnoconfig
+[ -z "$HOSTCC" ] && HOSTCC="$CC"
+
+make V=1 LDFLAGS=-static HOSTLDFLAGS=-static \
+CFLAGS_busybox="$debugcflags -Wl,-z,muldefs -Werror-implicit-function-declaration" \
```

Comment:
copying all the CFLAGS over seems a bit excessive. can your issue be fixed by just adding HOSTCC to the stanza ?

Reply:
Yes, fixed, thanks